current-work/README.md

```diff
@@ -0,0 +1,48 @@
+# The JSON Schema Organization's areas of focus or interest
+
+## About
+### What is this space?
+
+During an [Open Community Working Meeting](https://github.com/json-schema-org/community/issues/244), it was suggested that we could have a place for the core team (and maybe active community members?) to log their current areas of focus or interest in the form of GitHub Issues, Pull Requests, and Discussions.
+
+### Why?
+
+We concluded there was a LOT of activity taking place, and it was hard to get people to focus on the "important things". It is indeed hard to focus on priority items when there are many new GitHub Issues and Discussions each week, especially for those who are community members or not working on JSON Schema full or even part time. It was suggested that by asking for one to three items of priority, we might find it easier to draw attention where we feel it is needed.
+
+### How?
+
+The process will initially be manual and require people to commit to the adjacent file which contains the list.
+The list will be used as it is found on Friday to help assist populate the following Monday's Open Community Working Meeting agenda.
+We may introduce automations to assist in this at a later point, or a medium that isn't storing data in a text file.
+
+#### How should the file be updated?
+
+Initially, it's expected that only those who are actively contributing should look at or add items to the list.
+To add items to the list, either make commits directly in this repo.
+If you'd like permission to commit to the repo, please do ask on our [Slack server](https://json-schema.org/slack) in the `#community` channel.
+
+If you're adding to the list for the first time, please copy the below template, and add your GitHub username in alphabetical order.
+
+<details>
+  <summary>Show/hide template</summary>
+
+```md
+  **Link:** [Link to Github Issue, Pull Request, or Discussion]<br/>
+  **What?:** [The most simplest basic explanation of what the thing is about. Think a single tweets length or less.]<br/>
+  **Why does it matter?:** [Pitch why people should care about this too. Why do you care about it?]<br/>
+  **What can we do?:** [What would you like us to do in relation to this thing?]<br/>
```

Comment:
Is there a situation where this is anything other than "look at it and respond"?  This question feels like unnecessary boilerplate.

Reply:
Well, it could be you want someone to review the PR, or maybe you just want thoughts so far. Maybe you're looking to tackle one last bit relating to a specific comment. If it's a Discussion, it might be a specific thread you want people to reply to. IDK. Assumptions bad.
If we find it not useful, we can remove it later.

Reply:
If I want something specific looked at I'll say so in the entry.  This line just feels like an annoyance and I don't plan to include it in my entries.  This is supposed to be the easiest possible process for us to draw attention however we need it.  I really don't like boilerplate - I'm not going to block the PR over it, but I don't like it.